README.md

```diff
@@ -35,6 +36,12 @@ This library includes a collection of building blocks to help build out your DDD
 
 <!-- /TOC -->
 
+## Installation
+
+```sh
+yarn add ddd-ts
```

Comment:
I would suggest keeping `npm install ddd-ts` here as that is the default package manager.